src/plugins/wmr/plugin.js

```diff
@@ -65,33 +66,41 @@ export default function wmrPlugin({ hot = true } = {}) {
 				before += `const module={${hot ? 'hot:import.meta.hot' : ''}};\n`;
 			}
 
+			const hasExport = code.match(/\bexport\b/);
```

Comment:
We can reuse the ESM_KEYWORDS regex from [here](https://github.com/preactjs/wmr/blob/7ede32763b18cd56b29a10cf79e884469c3a052d/src/plugins/fast-cjs-plugin.js#L3):

```suggestion
			const hasExport = code.match(/(\bimport\s*(\{|\s['"\w_$])|[\s;]export(\s+(default|const|var|let)[^\w$]|\s*\{))/);
```